esm/interpreter/micropython.js

```diff
@@ -1,5 +1,6 @@
 import { fetchFiles, fetchJSModules, fetchPaths, stdio, writeFile } from './_utils.js';
 import { registerJSModule, run, runAsync, runEvent } from './_python.js';
+import mipSrc from './mip_src.py'     
```

Comment:
I am not really sure this is the way as it's not standard ECMAScript and it binds the project to rollup magic ... import attributes is likely what we're after and I wonder if we should use these instead and see how that goes with rollup? I understand this file *must* be somehow part of the bundle yet I'd rather copy it over to the dist folder and fetch it **only** when `config.packages` is present, eventually bootstrapping it faster any other time by resolving the fetch content once ... this also avoids the need for TextEncoder as the fetch can return directly a buffer ... what do you think? I could write some example code if needed.

Reply:
answering myself as we don't have a dist folder in here but I still think using non standard ways to import stuff breaks the module for those not using *rollup* ... this module can be installed and used in other projects so I need to think how to do best the same without breaking everything else that worked to date ... I might do a follow up MR with all changes so you can have a look and tell me if that works for you. I will reuse both the `.py` file and your smoke test + I will add integration tests but I really would like for this module to work in a standard way and I have already ideas about that.

Reply:
BTW, the test in CI fails because indeed there's no file named `mip_src.py` ... I think I have a better idea/solution to this.